### ISSUE TITLE
Regression crash?: TypeError: 'NoneType' object is not subscriptable 

### ISSUE THREAD

**Issue by gastonmorixe** [ml-explore/mlx-examples#964]
After this commit: https://github.com/ml-explore/mlx-examples/commit/1003a8b2dd59a22255a6a6c9a20f9d41f9812fb5#r146019927

I have tried other Llama 3.1 models and they fail in the exact same way. Is this a regression from PR #956?

```fish
~ via 🐍 v3.12.4 (mlx-0-17-2) took 6s
❯ mlx_lm.generate  --temp 0.7  --model "mlx-community/Meta-Llama-3.1-8B-bf16"  --prompt "Hello I am a Sunny I know Ruby on Rails, this is an example of a model class:

    ```rb"
Fetching 9 files: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 111353.20it/s]
Traceback (most recent call last):
  File "/Users/gaston/.pyenv/versions/mlx-0-17-2/bin/mlx_lm.generate", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/gaston/.pyenv/versions/3.12.4/envs/mlx-0-17-2/lib/python3.12/site-packages/mlx_lm/generate.py", line 175, in main
    tokenizer.chat_template = metadata["chat_template"]
                              ~~~~~~~~^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
(mlx-0-17-2)

❯ pip list  | grep mlx
mlx                     0.17.2
mlx-lm                  0.18.0
(mlx-0-17-2)
```

Thank you for the amazing work!

**Comment by awni** [comment]
Sorry about that, looks like a bug for models that don't have a chat template. I'll send a fix shortly.

**Comment by gastonmorixe** [comment]
cool, thank you @awni

**Comment by sarthaksavvy** [comment]
how to solve this issue, still not fixed

**Comment by awni** [comment]
I just uploaded 0.18.1 which fixes this. You can do `pip install -U mlx-lm` and it should work.

Closing for now but feel free to comment if you are still having issues.

**Comment by sarthaksavvy** [comment]
Thanks it worked 👍 
I just need to remove and then reinstall mlx-lm
```
pip uninstall mlx-lm
pip install -U mlx-lm
```

**Comment by gastonmorixe** [comment]
Fixed in PR #965 🚀 ty
